railgun/config/base/tmp-bindings.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
can we re-use previous role.yaml ? I think we already fix adding the udp api group and operations in single one.yaml .

Reply:
Are you saying following same file generated using tools ? 
https://github.com/Kong/kubernetes-ingress-controller/blob/350d0da8062392c7570c9f7f40e3558a5885a772/deploy/manifests/base/rbac.yaml

I believe they are manually created as you did.

Reply:
`deploy/manifests/base/rbac.yaml` was written manually. `railgun/config/rbac/role.yaml` was generated from type annotations.

Reply:
Right. 
Assuming helm chart installation will consume the new one railgun/role.yaml 
The real challenge right now is how to proceed to remove duplicates.

Reply:
After searching around a bit more, I think we're good to remove the temp patch in favor of modifying the source manifest in ff2ca18309138e20688e841900dff71a2cdbffaf

The source hasn't changed since we created the railgun directory. Were it auto-generated, it'd likely have _some_ change since then.